packages/services/f-test-data-creator/src/clients/searchApi.js

```diff
@@ -0,0 +1,24 @@
+const axios = require('axios');
+const { getLanguageForTenant } = require('../configuration/tenants');
+
+module.exports = class SearchApiService {
+    constructor (configuration) {
+        this.getRestaurantByPostcodeUrl = configuration.Services.Search.BaseAddress;
+    }
+
+    async getRestaurantsByPostcode (tenant, postcode, timeout = 5000) {
```

Comment:
Include Async in function name

Reply:
shall I just add async to every function name?